storage-node/src/server.rs

```diff
@@ -105,12 +97,147 @@ pub async fn storage_node_serve(ip_addr: &str, port: u16) -> ParpulseResult<()>
     let routes = route.or(heartbeat).or(catch_all);
     let ip_addr: IpAddr = ip_addr.parse().unwrap();
     warp::serve(routes).run((ip_addr, port)).await;
+}
+
+pub async fn storage_node_serve(
```

Comment:
Do we have to explicitly match xxx to LRU or LRU-K for every type of data store...?

Reply:
The concrete types have to match when we initialize a variable. We can't assign a variable as LRU or LRUK at the same time. So we have to do this.

Reply:
No. I mean if it is possible to modularize repetitive code.